### PR TITLE
User Input for any of the buttons is now being added to the api query.

### PR DESCRIPTION
**Pull request by JLP-55**
If the user selects any of the three buttons, the relevant data will be added to the api query string.
The for loop on lines 41-47 handles this.

### CODE REVIEW

File: assets/js/script.js
Comment (JLP-55) [inline assets/js/script.js:1]: The change page function works to change the page, however, the page up and page down functions need code added to them. 
I was struggling with this, so feel free to add to this file any code that will enable the functions to work as intended, as well as any additional features you think should be implemented.

File: assets/js/script.js [hunk 1]
```diff
@@ -197,26 +232,36 @@ function searchDatabase () {
...
                 var pageNum = event.target.textContent;
+                keyWord = keyWord + "&page=" + pageNum;
+                appendSearchItems.innerHTML = "";
+                appendSearchItemsBtn.innerHTML = "";
+                console.log(keyWord);
+                searchDatabase();
             };
 
             // Function to increase page buttons shown by 10.
             function pageUp () {
```
Comment (JLP-55) [inline assets/js/script.js:245]: These two functions need code added to scroll through the page numbers in the event that there are more than 10 pages to scroll through. This is totally not a priority.

File: assets/js/script.js [hunk 2]
```diff
@@ -123,18 +123,38 @@ function wateringParametersAdd (event) {
...
 // This function handles the api call.
 function searchDatabase () {
-    // Clears the four values buttons from the screen.
+
+    // work on lines 128 - 135
+    console.log(userInput.value);
+    console.log(keyWord);
+    console.log(keyWordUserSearch);
+
+    // This if statement is attempting to reset the page only when the user is searching for something new after already previously searching something.
```
Comment (JLP-55) [inline assets/js/script.js:132]: I was unable to get the variable keyWord to clear without refreshing the page. The searchDatabase function will work as intended in all cases, so long as you refresh the page before searching again.
This is not ideal from the end users perspective however. A quick fix could be to just add a "clear search items button"

File: assets/css/style.css
Comment (JLP-55) [inline assets/css/style.css:1]: Alex you will have to put your styling in another style.css file I think, as there are some selectors that we are both using.
I have commented out all your work so it is easy to spot. It is all located below my own styling.

Comment (JLP-55) [reply assets/css/style.css:1]: Nevermind, This issue has been dealt with. I just created my own css file for the plant database page. Your code should be fine Alex.

Comment (abalsillie) [reply assets/css/style.css:1]: Thanks Jack, that makes sense, once CSS converted to Tailwind, might remove the need for any CSS files- will confirm soon

**Review by abalsillie — APPROVED** [review]

File: assets/js/script.js (hunk 1 above)
Comment (abalsillie) [reply assets/js/script.js:246]: Noted

**Pull request merged**